jib-core/src/main/java/com/google/cloud/tools/jib/frontend/BuildStepsRunner.java

```diff
@@ -178,7 +203,7 @@ public void buildImage(HelpfulSuggestions helpfulSuggestions)
     }
```

Comment:
We prob want to change `BuildImageStepsExecutionException` to `BuildStepsExecutionException`.